src/IndexUpdater.php

```diff
@@ -118,6 +103,34 @@ public function updateProperties(array $properties): void
         }
     }
 
+    public function deleteProperty(array $property): void
+    {
+        $name = $property['name'];
+        $parentClasses = $property['parentClasses'];
+        $type = $property['type'];
+
+        $index = $this->findIndex($parentClasses, $type);
+
+        $script = sprintf(
+            'ctx._source.remove(\'%s\');',
+            $name
+        );
+
+        try {
+            $this->executeUpdateQuery($index, $type, $script);
+        } catch (Throwable $e) {
+            throw new FailToRemovePropertyException(
+                sprintf(
+                    'by script: %s AND type: %s',
+                    $result,
```

Comment:
Maybe my brain has stopped working but where does this variable come from?

Reply:
Your brain seems to be working fine :) You are correct. Fixed now